Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:14-alpine
+ENV NODE_ENV=development
```

Comment:
Why is this needed?

Reply:
Since it is installing and building, it needs the development dependencies. If we were consuming a published npm artifact we would use production. This dockerfile can be cleaned up for production by using multi stage builds and stuff but it would be best if it didn't have to do the building

Reply:
It could be set to production in later steps but at least for the install it needs development.